math/igraph/Portfile

```diff
@@ -52,7 +52,8 @@ configure.args-append   -DUSE_CCACHE=OFF \
                         -DIGRAPH_USE_INTERNAL_GMP=OFF \
                         -DIGRAPH_USE_INTERNAL_LAPACK=OFF \
                         -DIGRAPH_USE_INTERNAL_PLFIT=ON \
-                        -DIGRAPH_OPENMP_SUPPORT=OFF
+                        -DIGRAPH_OPENMP_SUPPORT=OFF \
+                        -DIGRAPH_WARNINGS_AS_ERRORS=OFF # temporary fix, remove for 0.10.1
```

Comment:
are you sure you can add these type of comments in `configure.args-append` - I surely doubt it....

Reply:
Thank you for pointing out the problem. This is fixed now.